controllers/chaosimpl/utils/record.go

```diff
@@ -50,20 +51,17 @@ func (d *ContainerRecordDecoder) DecodeContainerRecord(ctx context.Context, reco
 	var pod v1.Pod
 	podId, containerName, err := controller.ParseNamespacedNameContainer(record.Id)
 	if err != nil {
-		// TODO: organize the error in a better way
-		err = NewFailToFindContainer(pod.Namespace, pod.Name, containerName, err)
+		err = errors.Wrapf(ErrContainerNotFound, "container %s not found", record.Id)
```

Comment:
It would also print pod namespace and name in the origin error message, how about appending it?

Reply:
I thought the `record.Id` contains them all.

Reply:
How about changing the message to "container with record id %s not found"?

You know `record.Id` is the combination of `namespace/name/containerName`, because you have knowledge about how `record.Id` is implemented.

Reply:
Good.